app/assets/javascripts/discourse/app/components/d-lightbox/body.hbs

```diff
@@ -0,0 +1,57 @@
+<div
```

Comment:
Another question I have, could have this been made in a `<DModal />` ? Instead of having to rebuild all the backdrop logic and other things.